diff --git a/node_modules/wc-compiler/src/jsx.d.ts b/node_modules/wc-compiler/src/jsx.d.ts
index 2dc9baa..a2cb857 100644
--- a/node_modules/wc-compiler/src/jsx.d.ts
+++ b/node_modules/wc-compiler/src/jsx.d.ts
@@ -3,12 +3,17 @@ type IsCSSStyleDeclaration<T> = T extends CSSStyleDeclaration ? string : T;
 
 // create a utility type to extract the attributes from any given element's DOM interface.
 type ElementAttributes<E extends HTMLElement> = {
-  // Extract all properties from the element, including inherited ones.
+  // Extract all properties from the element, including inherited ones
   [A in keyof E]?: E[A] extends (...args: any) => any ? any : IsCSSStyleDeclaration<E[A]>;
 } & {
   class?: string;
-};
-
+  // have to manage this manually, can't seem to get this from TypeScript itself (not sure if just skill issue? :D)
+  // https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1790
+  // it should be there per https://github.com/mdn/browser-compat-data/pull/21875
+  // https://github.com/ProjectEvergreen/wcc/issues/236
+  popovertarget?: string;
+  popovertargetaction?: 'show' | 'hide' | 'toggle';
+}
 // map each HTML tag to a union of its attributes and the global attributes.
 type IntrinsicElementsFromDom = {
   [E in keyof HTMLElementTagNameMap]: ElementAttributes<HTMLElementTagNameMap[E]>;
